src/components/footer/www/footer.scss

```diff
@@ -18,12 +18,16 @@
         dt {
             display: block;
             margin-bottom: 8px;
-            font-weight: bold;
         }
 
         dd {
             display: block;
             margin: 5px 0;
+            line-height: 1.2rem;
```

Comment:
We should update the `#footer .list .dl` font-size. Things look a bit large.

<img width="1110" alt="screen shot 2016-05-27 at 11 40 10 am" src="https://cloud.githubusercontent.com/assets/3409578/15613103/c6fbfb9a-23ff-11e6-999a-9e1695733863.png">

I propose `font-size: .8rem`
<img width="1091" alt="screen shot 2016-05-27 at 11 40 58 am" src="https://cloud.githubusercontent.com/assets/3409578/15613131/e8839ebc-23ff-11e6-8b9a-546808c0b473.png">